docker_control.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+export IMAGE_NAME='starryoss/sysadmin'
```

Comment:
maybe rename this `sysadmin-build`? I'd expect the `sysadmin` image to actually run sysadmin. Is there any way to actually run `sysadmin` in a docker container?